spec/acceptance/init_spec.rb

```diff
@@ -21,5 +21,15 @@
     describe process('puppet-summary') do
       its(:args) { is_expected.to match %r{\ -host\ 127.0.0.1\ -port\ 4321} }
     end
+    # see https://github.com/skx/puppet-summary/blob/master/API.md
+    it 'works on get calls' do
+      shell('curl -s http://127.0.0.1:4321/?accept=application/json') do |r|
+        expect(r.stdout).to match(%r{null})
+      end
+    end
+    describe curl_command('http://127.0.0.1:4321/?accept=application/json') do
```

Comment:
I'd expect this to work too.

```suggestion
    describe curl_command('http://127.0.0.1:4321/', headers: {'Accept' => 'application/json'}) do
```

Both should work according to the API docs.